maven-core/src/main/java/org/apache/maven/project/MavenProject.java

```diff
@@ -90,6 +92,9 @@
 public class MavenProject
     implements Cloneable
 {
+
+    private static final transient Logger LOGGER = LoggerFactory.getLogger( MavenProject.class );
```

Comment:
Can you explain why you make it transient?

Reply:
Please remove 'transient'